### ISSUE TITLE
Add a README.md to each of the examples subfolders describing meaning of the data and models built

### ISSUE THREAD

**Issue by LRParser** [deepchem/deepchem#631]
This issue is to see if authors agree with the approach of adding a README.md to each main folder under ~/deepchem/examples, e.g.:

~/deepchem/examples/pdbbind/README.md
~/deepchem/examples/nci/README.md

etc

They would then describe what the script run does in enough detail for a novice bioinformatician to understand the usefulness of the built model.

**Comment by patrickhop** [comment]
If @rbharath green lights it, i'm happy to make some of these contributions.

**Comment by rbharath** [comment]
Yes please! PRs welcome :)

**Comment by JimSEOW** [comment]
@LRParser In addition, it is important for me to know WHAT [in silico drug discovery] GAPs have yet to be addressed. 

e.g. click2drug.org provide classification of tools according to the roles played in drug discovery. 
We need this overview, along the preclinical and clinical value chain of drug discovery for a new user to deepchem to see what have been done, what have yet to be done, for those with deepchem methods, are these methods up-to-date to the latest thinking etc.

**Comment by LRParser** [comment]
Submitted a basic PR above. The README.md format is nice since when you're browsing on Github, you start to learn what the datasets are. Can definitely use some help on this one as I'm still a novice - I'm mostly working on learning the project by reading the academic papers and trying to map to the source / creating docs as I go

**Comment by rbharath** [comment]
Opening this out to contribution. To target this issue, pick a subfolder in examples (https://github.com/deepchem/deepchem/tree/master/examples). Add a `README.md` file to this folder (if it doesn't have one already) that explains:

- The source of the dataset
- Explains what the model built in the example does to the level where a novel bioinformatician can understand.

If you'd like to implement this PR, leave a comment here specifying the folder(s) you're adding the readme to, and open a [WIP] tracking your implementation-in-progress.

**Comment by SauravMaheshkar** [comment]
Can I work on this @rbharath ? I was thinking I'll start with the `binding_pockets` directory. This folder already contains a README, but no description of the model itself. I would start with that for now and then work on the `adme` example 

Also, I noticed that most of these examples still have the following **redundant** imports, which ideally should be removed:
```
from __future__ import print_function
from __future__ import division
from __future__ import unicode_literals
```

**Comment by rbharath** [comment]
Yes, go for it! Some of the examples aren't running, so I'd suggest checking that the example runs correctly and fixing it if it doesn't

**Comment by rbharath** [comment]
I think our readthedocs now have this information. Closing this old issue
